packages/ui/src/components/va-virtual-scroller/useVirtualScrollerSizes.ts

```diff
@@ -0,0 +1,78 @@
+import { PropType, ExtractPropTypes, ref, Ref, shallowRef, computed, watch, onMounted } from 'vue'
+
+import { useResizeObserver } from '../../composables'
+
+export const useVirtualScrollerSizesProps = {
+  horizontal: { type: Boolean, default: false },
+  itemSize: {
+    type: [Number, String] as PropType<string | number>,
+    default: 0,
```

Comment:
Does 0 even work?

Reply:
We calculate items size ourselves, so, this prop is optional.

Reply:
Let's not use it in demos and examples then.